styles.css

```diff
@@ -436,6 +437,10 @@ form {
 }
 
 @media screen and (min-width: 768px) {
+  body {
+    min-width: fit-content;
+  }
+
```

Comment:
-This code resolves page width bug as it moves from mobile version to desktop version.